pkgs/by-name/co/codec2/package.nix

```diff
@@ -18,12 +19,23 @@ stdenv.mkDerivation rec {
     hash = "sha256-69Mp4o3MgV98Fqfai4txv5jQw2WpoPuoWcwHsNAFPQM=";
   };
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [
+    cmake
+    buildPackages.stdenv.cc
```

Comment:
There should be a comment here why this is needed.

Reply:
pushed